packages/orchestration/src/exos/chain-hub.js

```diff
@@ -232,11 +231,12 @@ const ChainHubI = M.interface('ChainHub', {
     M.or(DenomDetailShape, M.undefined()),
   ),
   getDenom: M.call(BrandShape).returns(M.or(M.string(), M.undefined())),
-  makeChainAddress: M.call(AccountIdArgShape).returns(CosmosChainAddressShape),
+  makeChainAddress: M.call(M.string()).returns(AccountIdArgShape),
```

Comment:
This is a good narrowing of the param type. But the return type should not be widened. It is indeed `CosmosChainAddressShape`.

Reply:
done